### DISCUSSION TITLE
Pluto on binder - how to load packages

### DISCUSSION THREAD

**Discussion by alex-s-gardner** [fonsp/Pluto.jl#2025] [Questions about using Pluto]
I also posted this on the Julia Slack channel:
https://julialang.slack.com/archives/C017Z6Q3WS3/p1649112168186659

I'm very keen to share an interactive Pluto notebook using Binder but I'm unable to get registered packages to load using `Pkg.add()` without the server timing out. Has anyone found a way around this issue? I suspect I need to percompile packages but not sure what steps I need to take to do that. 

Thanks!

**Comment by fonsp** [comment]
Hey! It could be that you are adding large packages that require a lot of memory to precompile, like Flux or DiffEq. Binder can have a fairly small memory limit (depending on the provider, time of day, etc), causing a memory overload.

Can you share a notebook file?

**Reply by alex-s-gardner** [reply]
Hey @fonsp the notebook I was trying to run is this one:

[notebooks/ITS_LIVE_plot_point.jl x](https://github.com/alex-s-gardner/ItsLive.jl/blob/298cfd2a6dd474313327e446d75402eb2c9dc412/notebooks/ITS_LIVE_plot_point.jl)

but I never did figure it out how to get it to work on Binder.

**Reply by nathanrboyer** [reply]
I get stuck "Precompiling"/"Reconnecting" on Binder every time, even for the simplest notebooks.
![image](https://github.com/user-attachments/assets/8cc52c4f-27be-4e6c-9f3c-138b5f035f27)

I don't think I've ever once got a notebook running on Binder, which is strange for a service so baked into the UI.
